src/hotspot/cpu/x86/assembler_x86.cpp

```diff
@@ -13536,28 +13836,27 @@ void Assembler::movq(Register dst, int32_t imm32) {
 
 void Assembler::movsbq(Register dst, Address src) {
   InstructionMark im(this);
-  emit_int24(get_prefixq(src, dst),
-             0x0F,
-             (unsigned char)0xBE);
+  int prefix = get_prefixq(src, dst, true /* page1 */);
```

Comment:
We are not consistent in the comment is_map1, M0, page1 all refer to the same thing. Also some places there is no comment that the true is for is_map1.

Reply:
Thank you.  Naming is now consistent with APX spec naming (is_map1 for the bool argument and M0 for the bit name).  I added inline comment  /* is_map1 */ for calls that pass the boolean argument.